internal/model/release_values.go

```diff
@@ -0,0 +1,250 @@
+}
+type Admin struct {
+	Enabled     bool              `yaml:"enabled,omitempty"`
+	Annotations map[string]string `yaml:"annotations,omitempty"`
+	Spec        Spec              `yaml:"spec,omitempty"`
+}
+type Internals struct {
+	Enabled     bool              `yaml:"enabled,omitempty"`
+	Annotations map[string]string `yaml:"annotations,omitempty"`
+}
+type Services struct {
+	Default   Default   `yaml:"default,omitempty"`
+	Neo4J     Neo4J     `yaml:"neo4j,omitempty"`
+	Admin     Admin     `yaml:"admin,omitempty"`
+	Internals Internals `yaml:"internals,omitempty"`
+}
+
+type SecurityContext struct {
+	RunAsNonRoot        bool   `yaml:"runAsNonRoot,omitempty"`
+	RunAsUser           int    `yaml:"runAsUser,omitempty"`
+	RunAsGroup          int    `yaml:"runAsGroup,omitempty"`
+	FsGroup             int    `yaml:"fsGroup,omitempty"`
+	FsGroupChangePolicy string `yaml:"fsGroupChangePolicy,omitempty"`
+}
+type ReadinessProbe struct {
+	FailureThreshold int `yaml:"failureThreshold,omitempty"`
+	TimeoutSeconds   int `yaml:"timeoutSeconds,omitempty"`
+	PeriodSeconds    int `yaml:"periodSeconds,omitempty"`
+}
+type LivenessProbe struct {
+	FailureThreshold int `yaml:"failureThreshold,omitempty"`
+	TimeoutSeconds   int `yaml:"timeoutSeconds,omitempty"`
+	PeriodSeconds    int `yaml:"periodSeconds,omitempty"`
+}
+type StartupProbe struct {
+	FailureThreshold int `yaml:"failureThreshold,omitempty"`
+	PeriodSeconds    int `yaml:"periodSeconds,omitempty"`
+}
+type PrivateKey struct {
+	SecretName interface{} `yaml:"secretName,omitempty"`
+	SubPath    interface{} `yaml:"subPath,omitempty"`
+}
+type PublicCertificate struct {
+	SecretName interface{} `yaml:"secretName,omitempty"`
+	SubPath    interface{} `yaml:"subPath,omitempty"`
+}
+type TrustedCerts struct {
+	Sources []interface{} `yaml:"sources,omitempty"`
+}
+type RevokedCerts struct {
+	Sources []interface{} `yaml:"sources,omitempty"`
+}
+type Bolt struct {
+	PrivateKey        PrivateKey        `yaml:"privateKey,omitempty"`
+	PublicCertificate PublicCertificate `yaml:"publicCertificate,omitempty"`
+	TrustedCerts      TrustedCerts      `yaml:"trustedCerts,omitempty"`
+	RevokedCerts      RevokedCerts      `yaml:"revokedCerts,omitempty"`
+}
+type HTTPS struct {
+	PrivateKey        PrivateKey        `yaml:"privateKey,omitempty"`
+	PublicCertificate PublicCertificate `yaml:"publicCertificate,omitempty"`
+	TrustedCerts      TrustedCerts      `yaml:"trustedCerts,omitempty"`
+	RevokedCerts      RevokedCerts      `yaml:"revokedCerts,omitempty"`
+}
+type Ssl struct {
+	Bolt  Bolt  `yaml:"bolt,omitempty"`
+	HTTPS HTTPS `yaml:"https,omitempty"`
+}
+type Image struct {
+	ImagePullPolicy string `yaml:"imagePullPolicy,omitempty"`
+	CustomImage     string `yaml:"customImage,omitempty"`
+}
+type Metadata struct {
+	Annotations map[string]string `yaml:"annotations,omitempty"`
+}
+type Statefulset struct {
+	Metadata Metadata `yaml:"metadata,omitempty"`
+}
+type Env struct {
```

Comment:
empty struct ?

Reply:
they need to be empty as they represent a `map[string] string` YAML entry

Reply:
should we declare it like this instead of empty struct ?
type Env map[string]string